CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 0.52.0 / 2021-11-03
+
+* [CHANGE] Automatically add sharding `relabel_configs` to all scrape jobs in `additionalScrapeConfigs` if `shards > 1`. #4324
```

Comment:
I don't think users are required to know how sharding works internally, so maybe let's rewrite this to sth like:
```suggestion
* [CHANGE]  Extend sharding capabilities to `additionalScrapeConfigs`. #4324
```